.github/workflows/tests.yaml

```diff
@@ -28,14 +28,31 @@ jobs:
           0xf000d910b99722d201c6cf88eb7d1112b43475b9765b118f289b5d65d919000d
         PUBLISHER_PRIVATE_KEY: >-
           0xeaa964d1353b075ac63b0c5a0c1e92aa93355be1402f6077581e37e2a846105e
-      name: 'Run SDK tests'
-      run: 'pnpm run test:sdk'
+      name: '${{ matrix.job-name }}'
+      run: '${{ matrix.command }}'
     - if: 'failure()'
       name: 'Print local testnet logs on failure'
       run: 'cat ${{ runner.temp }}/local-testnet-logs.txt'
       shell: 'bash'
     timeout-minutes: 15
-name: 'Run the SDK tests'
+    strategy:
+      matrix:
+        include:
+        - job-name: 'Frontend: [unit]'
+          command: 'pnpm run test:frontend:unit'
+        - job-name: 'SDK: [unit]'
+          command: 'pnpm run test:sdk:unit'
+        - job-name: 'SDK: [e2e, base, general]'
+          command: 'pnpm run test:sdk:e2e:base-general'
+        - job-name: 'SDK: [e2e, base, websockets]'
+          command: 'pnpm run test:sdk:e2e:base-ws'
+        - job-name: 'SDK: [e2e, arena, general]'
+          command: 'pnpm run test:sdk:e2e:arena-general'
+        - job-name: 'SDK: [e2e, arena, websockets]'
+          command: 'pnpm run test:sdk:e2e:arena-ws'
+        - job-name: 'SDK: [e2e, arena, candlesticks]'
+          command: 'pnpm run test:sdk:e2e:arena-candlesticks'
```

Comment:
How about cleaning this up a little? Just for more tidy printouts in checks tab:

<img width="805" alt="Screenshot 2025-03-14 at 11 56 38" src="https://github.com/user-attachments/assets/f32b8ee7-9b53-49be-b0ec-140af6344130" />

Reply:
In fact I think you could even take out `job-name` and just remove the `name` field entirely, since you'll see the run command

Also, note that each `command` has `pnpm run test:`, so it might be worth using `command-suffix` so that the only think that shows up in the printout is, for example `frontend: unit`

Somethin like       `run: 'pnpm run test:${{ matrix.command-suffix }}'